lms/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
This really belongs on the enrollment API given that object we are querying is the enrollment.